integrationtest/TargetProjects/NetCore/Library.FSharp.XUnit/Library.FSharp.XUnit.fsproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
We're removing the FSharp code since is't not being worked on actively for a while, so this might result in merge conflicts.